### DISCUSSION TITLE
Getting started with Packetsender

### DISCUSSION THREAD

**Discussion by highendau** [dannagle/PacketSender#318] [General]
Hi Everyone, 

Before I sink my teeth into PacketSender, I just want to ask a few questions to see wether this application would be suitable for our purpose, as well as a questions on making donations due to being used for Commercial purposes. 

Firstly; 
I would like to install this on a recommended Linux instance on my Oracle Cloud and being able to remotely access via a browser using IP address & Port (xxx.xxx.xx.xxx:pppp). 
Can you advise wether this can be done, and what is the most recommended Linux OS for this? 

Secondly;
Can PacketSender be used for connecting IoT Devices using TCP connections? 

Thirdly; 
If the answer to the above is yes, is there a Storage and WebRTC option which can allow for data from the external connected devices to be stored (with automatic erase after a preset time), and recalled using a specific URL for each device? 

Fourthly; 
Is there a limit on how many devices can be onboard of 1 server?

Lastly; 
As this will be for commercial use, What is the best way to make payment(s)?

Thank you in advance, 
Michael

**Comment by dannagle** [comment]
Hello Michael,

Thanks for the interest. I will try to answer you questions.

1. being able to remotely access via a browser.....  what is the most recommended Linux OS for this?
A-1: Packet Sender does not have a web UI. I am not sure it will do what you want. Perhaps you are looking at a virtual desktop.
A-2: My favorite Linux distro is Ubuntu. I use it everywhere, including compiling the AppImage. 


2. be used for connecting IoT Devices using TCP connections?
A: Not sure what you mean. Packet Sender can connect to an IoT device and send it a command through UDP, TCP, SSL, or GET/POST request.

3. Storage and WebRTC option....
A: It's configuration settings do not go that deep. It stores UDP, TCP, SSL, and GET/POST requests and allows recalling them to send out. 

4. Is there a limit on how many devices can be onboard of 1 server?
A: No limit to stored packets. No limit to bound servers other than what the OS forces... which is roughly 65000.

5. What is the best way to make payment(s)?
A-1:  You can sponsor [through GitHub](https://github.com/sponsors/dannagle) or donate [through PayPal](https://www.paypal.com/biz/fund?id=RGDM593DYR9LJ)


I hope that helps,
Dan
